Session.vim

```diff
@@ -0,0 +1,60 @@
+let SessionLoad = 1
```

Comment:
please don't add any not relate file !!!!!!!!!

Reply:
Yes thank you. I am trying to clean this up right now.